app/browser/api/ledger.js

```diff
@@ -95,6 +99,27 @@ const clientOptions = {
   environment: process.env.LEDGER_ENVIRONMENT || 'production'
 }
 
+var platforms = {
+  'darwin': 'osx',
+  'win32x64': 'winx64',
+  'win32ia32': 'winia32',
+  'linux': 'linux'
+}
+
+let platform = platforms[process.platform]
+
+if (process.platform === 'win32') {
+  platform = platforms[process.platform + process.arch]
+}
+
+let referralServer = 'https://laptop-updates-staging.herokuapp.com'
```

Comment:
Privacy note: We are trying to not add any more 3rd party requests in Brave (https://github.com/brave/browser-laptop/issues/12632). For production, please make sure this endpoint is a brave.com subdomain, assigning a CNAME if needed.

cc @aekeus